### ISSUE TITLE
Default branch fallback fails

### ISSUE THREAD

**Issue by trieloff** [adobe/helix-resolve-git-ref#287]
```bash
$ http "https://adobeioruntime.net/api/v1/web/helix-pages/helix-services/resolve-git-ref@latest?owner=trieloff&repo=test"
HTTP/1.1 400 Bad Request
Access-Control-Allow-Headers: Authorization, Origin, X-Requested-With, Content-Type, Accept, User-Agent
Access-Control-Allow-Methods: OPTIONS, GET, DELETE, POST, PUT, HEAD, PATCH
Access-Control-Allow-Origin: *
Connection: keep-alive
Content-Length: 106
Content-Type: application/json
Date: Tue, 06 Oct 2020 12:41:09 GMT
Strict-Transport-Security: max-age=7884000; includeSubDomains
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-Request-ID: o1z38fqIQMqdk0V8Lrj1sW21vjOzRAGO
X-XSS-Protection: 1; mode=block
x-openwhisk-activation-id: 87459b97b2a54f65859b97b2a53f657c

{
    "code": "o1z38fqIQMqdk0V8Lrj1sW21vjOzRAGO",
    "error": "There was an error processing your request."
}
```

**Comment by trieloff** [comment]
```bash
$  wsk activation get c509b552ad5445ba89b552ad54b5bada
ok: got activation c509b552ad5445ba89b552ad54b5bada
{
    "namespace": "helix",
    "name": "resolve-git-ref@ci1727",
    "version": "0.0.1",
    "subject": "helix",
    "activationId": "c509b552ad5445ba89b552ad54b5bada",
    "start": 1601989411648,
    "end": 1601989411976,
    "duration": 328,
    "statusCode": 0,
    "response": {
        "status": "action developer error",
        "statusCode": 0,
        "success": false,
        "result": {
            "error": "The action did not produce a valid response and exited unexpectedly."
        }
    },
    "logs": [],
    "annotations": [
        {
            "key": "path",
            "value": "helix/helix-services/resolve-git-ref@ci1727"
        },
        {
            "key": "waitTime",
            "value": 24
        },
        {
            "key": "kind",
            "value": "nodejs:10"
        },
        {
            "key": "timeout",
            "value": false
        },
        {
            "key": "limits",
            "value": {
                "concurrency": 200,
                "logs": 10,
                "memory": 256,
                "timeout": 60000
            }
        },
        {
            "key": "initTime",
            "value": 195
        }
    ],
    "publish": false
}
```

**Comment by stefan-guggisberg** [comment]
fixed via #289
